graph_nets/demos_tf2/graph_nets_basics.ipynb

```diff
@@ -703,85 +772,37 @@
     },
     {
       "cell_type": "code",
-      "execution_count": 0,
       "metadata": {
-        "colab": {},
         "colab_type": "code",
-        "id": "Yf8d8qV2gQV8"
+        "id": "Yf8d8qV2gQV8",
+        "colab": {}
       },
-      "outputs": [],
       "source": [
-        "tf.reset_default_graph()\n",
         "graphs_tuple_tf = utils_tf.data_dicts_to_graphs_tuple(graph_dicts)\n",
-        "\n",
-        "with tf.Session() as sess:\n",
-        "  graphs_tuple_np = sess.run(graphs_tuple_tf)\n",
-        "\n",
-        "plot_graphs_tuple_np(graphs_tuple_np)"
-      ]
+        "plot_graphs_tuple_np(graphs_tuple_tf)"
```

Comment:
I think this method is now being called with for `GraphTuples` containing tensors, and `GraphTuples` containing numpy arrays. While this seems to work, it can be confusing. So maybe just rename it to `plot_graphs_tuple`, and add a line in the method saying that this method should work for both `GraphsTuple`s containing tensors, or numpy arrays.